.github/workflows/crowdin.yml

```diff
@@ -13,6 +13,7 @@ on:
 
 jobs:
   sync-crowdin:
+    if: github.repository_owner == 'revanced'
```

Comment:
Why is this necessary? 

It is expected for workflows to fail if you don't set the required secrets, and forkers have to explicitly enable workflows in the first place

Reply:
It would be spamming status failed in `actions` if forker enables workflows for testing, I can revert the changes if you like